dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -303,7 +305,7 @@ fi
 
 COMMAND_DIR=$(dirname "$0")
 
-if [ "${CHE_MULTI_USER}" == "true" ]; then
+if [ "${CHE_MULTIUSER}" == "true" ]; then
```

Comment:
+1 to sync this variable name and variable name we are already using in Che server/CLI